kanidm_book/src/installing_the_server.md

```diff
@@ -10,18 +10,26 @@ You can fetch these with:
     docker pull kanidm/server:latest
     docker pull kanidm/radius:latest
 
+If you wish to use an x86\_64 cpu-optimised version, you should use:
+
+    docker pull kanidm/server:x86_64_latest
+
+You may need to adjust your example commands throughout this document to suit.
+
 There are a number of configuration steps you must perform before you can run this container.
 
 ## System Requirements
 
 ### CPU
 
-You must have a CPU that is from 2013 or newer (Haswell, Ryzen). This is due to the usage of
-SIMD for a number of internal components.
+If you are using the cpu-optimised version, you must have a CPU that is from 2013 or newer
+(Haswell, Ryzen). This is due to the usage of SIMD for a number of internal components.
 
```

Comment:
Based on [Intel Wiki](https://en.wikipedia.org/wiki/List_of_Intel_Pentium_processors#Comet_Lake_based_Pentiums) , It is safe to say that all Intel Celeron and Pentium-branded cpu are not supported. Even the current one that are released this year/even this month.

Maybe add which SIMD instructions are important. I think important one is `AVX2`.

Better highlight these points make it easier to making the right choice between the two versions.

Reply:
I'll list the exact flags that are used :)